metadata/dashboard/dashboard_metadata.go

```diff
@@ -923,22 +927,44 @@ func (m *MetadataServer) GetMetadataList(c *gin.Context) {
 
 }
 
+func (m *MetadataServer) GetSearch(c *gin.Context) {
+	query := c.Param("query")
+	searchParameters := &api.SearchCollectionParams{
+		Q:       query,
+		QueryBy: "Name",
+	}
+	result, err := typesenseClient.Collection("resource").Documents().Search(searchParameters)
+	if err != nil {
+		c.JSON(500, "Failed to fetch resources")
+	}
+	c.JSON(200, result)
+}
+
 func (m *MetadataServer) Start(port string) {
 	router := gin.Default()
 	router.Use(cors.Default())
 
 	router.GET("/data/:type", m.GetMetadataList)
 	router.GET("/data/:type/:resource", m.GetMetadata)
+	router.GET("/search/:query", m.GetSearch)
 
 	router.Run(port)
 }
 
 func main() {
+	logger := zap.NewExample().Sugar()
 	metadataHost := os.Getenv("METADATA_HOST")
 	metadataPort := os.Getenv("METADATA_PORT")
+	typesenseHost := os.Getenv("TYPESENSE_HOST")
```

Comment:
Add new env vars into the helm charts. Otherwise the branches get out of sync

Reply:
added to dashboard_metadata chart